configs/benchmark-dsign.json

```diff
@@ -0,0 +1,108 @@
+{
+  "description": "aws 30 nodes on 3 shard in one region",
+  "client": {
+    "num_vm": 0,
+    "type": "t3.medium",
+    "regions": "iad"
+  },
+  "leader": {
+    "num_vm": 3,
+    "type": "t3.small",
+    "regions": "iad",
+    "protection": false,
+    "root": 15
+  },
+  "explorer_node": {
+    "num_vm": 3,
+    "type": "t3.small",
+    "regions": "iad",
+    "root": 25,
+    "protection": false
+  },
+  "bootnode": {
+     "enable": true,
+     "server": "52.40.84.2",
+     "name": "b6.harmony.one",
+     "port": 9890,
+     "user": "ec2-user",
+     "key": "oregon-key-benchmark.pem",
+     "p2pkey": "b2-9870.key",
+     "log_conn": false
+  },
+  "bootnode1": {
+     "enable": true,
+     "server": "54.86.126.90",
+     "name": "b5.harmony.one",
+     "port": 9890,
+     "user": "ec2-user",
+     "key": "virginia-key-benchmark.pem",
+     "p2pkey": "b1-9870.key",
+     "log_conn": false
+  },
+  "azure": {
+    "num_vm": 0,
+    "regions": [
+      "eastus",
+      "westeurope",
+      "southeastasia"
+    ]
+  },
+  "benchmark": {
+    "shards": 3,
```

Comment:
why do we need 3 shard for double signing?  maybe 1 shard is good enough for testing.